examples/ethernet/ethernet/main/ethernet_main.c

```diff
@@ -32,92 +32,45 @@
 #include "tcpip_adapter.h"
 #include "nvs_flash.h"
 #include "driver/gpio.h"
-#include "tlk110_phy.h"
 
-static const char *TAG = "eth_demo";
+extern void phy_device_check_phy_init(void);
```

Comment:
Here is the latest suggestion.  lan8720_phy.c has:

```
    void phy_update_config(eth_config_t *config);
```

and ethernet_main has:

```
    eth_config_t config;
    config.gpio_config = eth_gpio_config_rmii;
    config.tcpip_input = tcpip_adapter_eth_input;
    config.phy_power_enable = phy_device_power_enable;
    phy_device_update_config(&config);
    ret = esp_eth_init(&config);

```